BelatrixConnect_AutomationDSL/src/test/java/com/belatrix/connect/framework/DSL.java

```diff
@@ -13,9 +15,45 @@ public DSL(WebDriver driver) {
 
 	/**
 	 * @param locator
-	 */
+	 **/
```

Comment:
por favor coloca que cosa es lo que realiza este metodo, sencillamente coloca "this method make click in the locator parameter"